src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -807,23 +807,19 @@ internal sealed partial class MonoSDBHelper
 
         internal readonly ILogger logger;
 
-        [GeneratedRegex(@"\<(?<varName>[^)]*)\>(?<varId>[^)]*)(__)(?<scopeId>\d+)", RegexOptions.Singleline)]
-        private static partial Regex RegexForAsyncLocals(); //<testCSharpScope>5__1 // works
+#pragma warning disable SYSLIB1045
+        private static Regex regexForAsyncLocals = new Regex(@"\<(?<varName>[^)]*)\>(?<varId>[^)]*)(__)(?<scopeId>\d+)", RegexOptions.Singleline);
```

Comment:
```suggestion
        private static Regex s_regexForAsyncLocals = new(@"\<(?<varName>[^)]*)\>(?<varId>[^)]*)(__)(?<scopeId>\d+)", RegexOptions.Singleline);
```

.. and similar for all the other instances.